.github/workflows/plugin_install.yml

```diff
@@ -0,0 +1,71 @@
+name: Plugin Install
+
+on: [push, pull_request, workflow_dispatch]
+
+env:
+  OPENSEARCH_VERSION: "1.3.0"
+
+jobs:
+  plugin_install:
+    name: Plugin Install
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        jdk: [8, 11, 14]
+
+    steps:
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.jdk }}
+
+    - name: Checkout security
+      uses: actions/checkout@v2
+
+    - name: Build
+      run: ./gradlew clean assemble -Dbuild.snapshot=false
+
+    - name: Download OpenSearch Core
+      run: |
+        wget https://ci.opensearch.org/ci/dbc/distribution-build-opensearch/${{env.OPENSEARCH_VERSION}}/latest/linux/x64/builds/opensearch/dist/opensearch-min-${{env.OPENSEARCH_VERSION}}-linux-x64.tar.gz
+        tar -xzf opensearch-*.tar.gz
+        rm -f opensearch-*.tar.gz
+
+    - name: Move and rename security plugin for installation
+      run: mv build/distributions/opensearch-security-${{env.OPENSEARCH_VERSION}}.0.zip opensearch-security.zip
+
+    - name: Run OpenSearch with plugin
+      run: |
+        cat > os-ep.sh <<EOF
+        yes | opensearch-plugin install file:///docker-host/security-plugin.zip
+        chmod +x plugins/opensearch-security/tools/install_demo_configuration.sh
+        yes | plugins/opensearch-security/tools/install_demo_configuration.sh
+        echo "plugins.security.unsupported.restapi.allow_securityconfig_modification: true" >> /opensearch/config/opensearch.yml
```

Comment:
Why is this needed?

Reply:
I'm not sure if it is, this is copy and pasted from the other codebase.  I'll do an experiment if this can be removed.

Reply:
I've removed it and the tests pass, done!